docs/react-component-atlas.md

```diff
@@ -74,8 +74,8 @@ This is a high-level overview of the structure of the React component tree that
 > > > [`<MultiFilePatchController>`](/lib/controllers/multi-file-patch-controller.js)
 > > > [`<MultiFilePatchView>`](/lib/views/multi-file-patch-view.js)
 > > >
-> > > Render a sequence of git-generated file patches within a TextEditor, using decorations to include contextually
-> > > relevant controls.
+> > > Render a sequence of git-generated file patches within a TextEditor, using decorations to include contextually relevant controls.
+> > > See [`MultiFilePatchView` atlas](#multifilepatchview-atlas) below for a more detailed breakdown.
```

Comment:
What I _really_ want is a tool that parses our source and outputs this atlas as an .svg diagram or something 🤔